website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster.sh

```diff
@@ -5,7 +5,7 @@ kind: ClusterConfig
 metadata:
   name: ${CLUSTER_NAME}
   region: ${AWS_DEFAULT_REGION}
-  version: "1.21"
+  version: "1.22"
```

Comment:
this is a good change, but it is out of scope of this PR

Reply:
True. I'll add it to a separate PR.